src/NinjaTools.FluentMockServer.API/Services/ExpectationService.cs

```diff
@@ -173,20 +169,14 @@ public async ValueTask<long> PruneAsync(CancellationToken token = default)
                 return null;
             }
         }
-        
+
         public async IAsyncEnumerable<Expectation> SeedAsync(int count, [EnumeratorCancellation] CancellationToken token = default)
         {
-#if DEBUG
             var expectations = ExpectationFactory.FakeExpectations(count);
             await _context.Expectations.AddRangeAsync(expectations, token);
             await _context.SaveChangesAsync(token);
 
-            await foreach (var expectation in _context.Expectations.AsAsyncEnumerable())
-            {
-                yield return expectation;
-            }
-#endif
+            await foreach (var expectation in _context.Expectations.AsAsyncEnumerable()) yield return expectation;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)